A patch that fakes the android version so that NNAPI can be loaded and used
by tensorflow on chrome OS.

This patch can be removed when b/140896675 is fixed upstream and backported.


diff --git a/tensorflow/lite/nnapi/nnapi_implementation.cc b/tensorflow/lite/nnapi/nnapi_implementation.cc
index 71a4de53..1f8c7df9 100644
--- a/tensorflow/lite/nnapi/nnapi_implementation.cc
+++ b/tensorflow/lite/nnapi/nnapi_implementation.cc
@@ -105,8 +105,9 @@ int ASharedMemory_create(const char* name, size_t size) {

 const NnApi LoadNnApi() {
   NnApi nnapi = {};
-  nnapi.android_sdk_version = 0;
-
+  // TODO(slangley): Hardcode android sdk version until b/140896675 is resolved.
+  // nnapi.android_sdk_version = 0;
+  nnapi.android_sdk_version = 27;
 #ifdef __ANDROID__
   nnapi.android_sdk_version = GetAndroidSdkVersion();
   if (nnapi.android_sdk_version < 27) {
